dbcron/update_db.py

```diff
@@ -6,6 +6,7 @@
 import redis
 
 load_dotenv()
+print("this is running")
```

Comment:
Let's get rid of unused or unnecessary prints.  If it's a log, make it clear with something like `[LOG]` before or something like that.